pkg/common/common.go

```diff
@@ -283,3 +285,28 @@ func GetManifestArtifactType(manifestContent ispec.Manifest) string {
 
 	return manifestContent.Config.MediaType
 }
+
+func AddExtensionSecurityHeaders(h http.Handler) http.HandlerFunc { //nolint:varnamelen
```

Comment:
Could this function also return a mux.MiddlewareFunc to be applied on the router (mgmtRouter for eg)? does it make sense to be treated as a mdw as well ?